testdata/scripts/ldflags.txt

```diff
@@ -1,7 +1,7 @@
 # Note the proper domain, since the dot adds an edge case.
 env GOPRIVATE=domain.test/main
 
-garble build -ldflags='-X=main.unexportedVersion=v1.0.0 -X=domain.test/main/imported.ExportedVar=replaced'
+garble -tiny build -ldflags='-X=main.unexportedVersion=v1.0.0 -X=domain.test/main/imported.ExportedVar=replaced'
```

Comment:
keep this as it was and just add another exec with `-tiny` in the non-short mode, like some of the other tests.

testing `-ldflags` without `-tiny` is also important.

Reply:
Not relevant now, short names are now in all modes.

Reply:
I don't think that matters. We should still test ldflags without tiny, at the very least. If you want to add an extra test with tiny, that's fine.